src/blueapi/utils/__init__.py

```diff
@@ -7,7 +7,6 @@
 __all__ = [
     "handle_all_exceptions",
     "load_module_all",
-    "ConfigLoader",
```

Comment:
Should: Is there any typing-related reason for this change. If not it is unrelated and should be reverted.

Reply:
Pyright was complaining. Pyright complains about more things than just typing

Reply:
ConfigLoader doesn't appear to be imported, so I expect it was because it's not in the namespace.

Reply:
yes, pyright complained about the import wrong, so deleting the import meant that the export also needs to be deleted